README.md

```diff
@@ -106,6 +106,50 @@ Some example patterns with behavior are listed:
 
 Old versions of Samsung Android browsers are incompatible with formatting. These will not attempt to intercept the values and format the input.
 
+## Development
+
+### Running the demo app
+
+We use node 10 in development. To run a demo app on port 3099, run the following:
+
+```sh
+npm install
+npm run development
+```
+
+### Running unit tests
+
+The following command will run the linting task and the unit tests.
+
+```sh
+npm run test
+```
+
+### Running integration tests
+
+First, [sign up for a free open source Sauce Labs account](https://saucelabs.com/open-source).
+
+Then, create a .env file.
```

Comment:
I tend to have a strong preference for including a `.env.example` file that one can `cp .env.example .env` and would already include the body of

```
SAUCE_USERNAME=your-user-name
SAUCE_ACCESS_KEY=your-access-key
```